server/app.js

```diff
@@ -1,17 +1,23 @@
 import express from 'express';
 import logger from 'morgan';
 import helmet from 'helmet';
-import meetupAPI from './routes/meetup';
-import questionAPI from './routes/question';
-import rsvpAPI from './routes/rsvp';
+import { config } from 'dotenv';
+// import meetupAPI from './routes/meetup';
```

Comment:
avoid having unused commented code

Reply:
Okay, I will. thanks for pointing that out